dvc/config.py

```diff
@@ -143,6 +155,13 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
     SECTION_CORE_ANALYTICS = "analytics"
     SECTION_CORE_ANALYTICS_SCHEMA = And(str, is_bool, Use(to_bool))
 
+    SECTION_HASH = "hash"
```

Comment:
Should we also call it `checksum` section btw? It feels like we should stick with `hash` or `checksum` in the code. Checksum feels like a better fit IMHO.

Reply:
Yeah, I like checksum, too.